src/dataviews/category-dataview-model.js

```diff
@@ -44,6 +45,7 @@ module.exports = DataviewModelBase.extend({
     this._searchModel = new SearchModel();
 
     this.on('change:column change:aggregation change:aggregation_column', this._reloadMap, this);
+    this.on('change:prefix change:suffix', this._reloadMap, this);
```

Comment:
After talking with @javierarce I understand that these are not needed for maps API or dataviews, so extracted them up to DI.